conformance/packages/conformance-tests/src/name_server/rfc8906.rs

```diff
@@ -9,9 +9,15 @@ mod basic;
 mod extended;
 
 fn setup() -> Result<(Network, NameServer<Running>, Client)> {
+    setup_with_sign_settings(SignSettings::default())
+}
+
+fn setup_with_sign_settings(
```

Comment:
Suggest moving `setup()` to `SignSettings::setup(self)` to make this a little nicer.

Reply:
Ah, fair enough!